app/assets/stylesheets/application/member-card.scss

```diff
@@ -1,16 +1,34 @@
+/* Uso de Flexbox para organizar el contenedor de los cards*/
+.row.to-member-cards {
+  display: flex;
+  flex-wrap: wrap; //Crea una nueva fila cuando se llena la segunda columna
+  align-items: stretch; // En la fila asegura que todas las columnas tengan la misma altura.
+}
+
+/* Configura las columnas para usar Flexbox y organizar el contenido verticalmente */
+.col-lg-6.col-sm-12 {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
this is bad because you are changing a [bootstrap](https://getbootstrap.com/) class and maybe it can affect to other elements, it would be better to use a new class name. Nevertheless as we are using bootstrap probably you could use the flex class and the flex direction from the framework on that specific block